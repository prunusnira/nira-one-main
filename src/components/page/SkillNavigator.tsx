import Button from "../button";
import { Wrapper, Title, TitleSub, BWSpan, WBSpan } from "../common.style";
import Item from "../item/item";

type Props = {
    bgImage: string;
};

const SkillNavigator = ({ bgImage }: Props) => {
    return (
        <Item bgImage={bgImage}>
            <Wrapper>
                <Title>
                    <BWSpan>Skill Navigator</BWSpan>
                </Title>
                <TitleSub>
                    <WBSpan>Personal Data Manager for GITADORA</WBSpan>
                </TitleSub>
                <Button
                    src="https://sin.nira.one"
                    text="GO TO SKILL NAVIGATOR"
                />
            </Wrapper>
        </Item>
    );
};

export default SkillNavigator;
