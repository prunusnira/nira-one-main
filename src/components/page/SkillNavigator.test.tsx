import { render } from "@testing-library/react";
import SkillNavigator from "./SkillNavigator";

describe("banpicker section", () => {
    it("snapshot test", () => {
        const wrapper = render(<SkillNavigator bgImage="" />);
        expect(wrapper).toMatchSnapshot();
    });

    it("render test", () => {
        const wrapper = render(<SkillNavigator bgImage="" />);
        wrapper.getByText(/Skill Navigator/);
    });
});
