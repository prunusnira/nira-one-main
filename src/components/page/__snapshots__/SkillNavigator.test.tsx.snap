// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banpicker section snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="target css-1hzir52"
      >
        <div
          class="css-15z58y6"
        >
          <div
            class="css-h2mica"
          >
            <span
              class="css-1d7eyad"
            >
              Skill Navigator
            </span>
          </div>
          <div
            class="css-wyy442"
          >
            <span
              class="css-1fqqz26"
            >
              Personal Data Manager for GITADORA
            </span>
          </div>
          <div
            class="css-1w7pkgh"
          >
            <button
              class="css-2j4jox"
            >
              GO TO SKILL NAVIGATOR
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="target css-1hzir52"
    >
      <div
        class="css-15z58y6"
      >
        <div
          class="css-h2mica"
        >
          <span
            class="css-1d7eyad"
          >
            Skill Navigator
          </span>
        </div>
        <div
          class="css-wyy442"
        >
          <span
            class="css-1fqqz26"
          >
            Personal Data Manager for GITADORA
          </span>
        </div>
        <div
          class="css-1w7pkgh"
        >
          <button
            class="css-2j4jox"
          >
            GO TO SKILL NAVIGATOR
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
